v2/apiserver/config_test.go

```diff
@@ -303,21 +303,65 @@ func TestSessionsServiceConfig(t *testing.T) {
 			},
 		},
 		{
-			name: "ROOT_USER_PASSWORD required but not set",
+			name: "ROOT_USER_SESSION_TTL required but not set",
 			setup: func() {
 				os.Setenv("OIDC_ENABLED", "true")
 			},
+			assertions: func(err error) {
+				require.Error(t, err)
+				require.Contains(t, err.Error(), "value not found for")
+				require.Contains(t, err.Error(), "ROOT_USER_SESSION_TTL")
+			},
```

Comment:
See [this comment](https://github.com/brigadecore/brigade/pull/1308/files#r585662345)

In the adjustment I proposed there, there's a default value, so it you take that suggestion, these assertions need to be adjusted accordingly.

Reply:
Same for OIDC user session TTL.

Reply:
Updated, thanks!